src/libraries/Microsoft.Extensions.FileProviders.Physical/tests/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <Import Project="..\..\Directory.Build.props" />
+  <PropertyGroup>
+    <IsAspNetCoreApp>true</IsAspNetCoreApp>
```

Comment:
why do we need this on a test project?

Reply:
not sure if needed, I just wanted to prevent the test project from using `..\Directory.Build.props` (the one in the parent folder) because the tag `UnsupportedOSPlatforms` was causing a build issue only on net461 on this project. `IsAspNetCoreApp` was part of the parent Directory.Build.props so I included it here.
This was the error: https://github.com/dotnet/runtime/pull/55664/checks?check_run_id=3081962573
```
src\libraries\System.Private.CoreLib\src\System\Runtime\Versioning\PlatformAttributes.cs(35,48): 
error CS0436: (NETCORE_ENGINEERING_TELEMETRY=Build) The type 'OSPlatformAttribute' in 'D:\workspace\_work\1\s\src\libraries\System.Private.CoreLib\src\System\Runtime\Versioning\PlatformAttributes.cs'
conflicts with the imported type 'OSPlatformAttribute' in 'Microsoft.Extensions.FileProviders.Physical, Version=6.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60'. 
Using the type defined in 'D:\workspace\_work\1\s\src\libraries\System.Private.CoreLib\src\System\Runtime\Versioning\PlatformAttributes.cs'.
```

Reply:
Something is going on with the way we include the platform attributes:

https://github.com/dotnet/runtime/blob/405da674b633a99cc41be9e18c2e4d84b1b01639/src/libraries/Directory.Build.targets#L249

@jeffhandley @buyaa-n it seems like we shouldn't honor this flag for test projects at all?

Reply:
> Something is going on with the way we include the platform attributes:

I don't think that is the issue, but I don't know why the test project is referencing 2 versions of PlatformAttributes.cs, `5.0+ builds` should use public `PlatformAttributes` and other `builds < 5.0` should reference internal `PlatformAttributes` , seems somehow it is loaded both on `build < 5.0`, guess @Jozkee need to double check the references

Reply:
Ah I know why it is. It is because this P2P:
https://github.com/dotnet/runtime/blob/ebb032688b393d184620e2b6de0ca9a5d544e853/src/libraries/Microsoft.Extensions.FileProviders.Physical/tests/Microsoft.Extensions.FileProviders.Physical.Tests.csproj#L21

And the source project has internals visible to the test:
https://github.com/dotnet/runtime/blob/ebb032688b393d184620e2b6de0ca9a5d544e853/src/libraries/Microsoft.Extensions.FileProviders.Physical/src/Properties/InternalsVisibleTo.cs#L6

You can fix this by adding `IncludePlatformAttributes=false` to the test project, rather than adding a `Directory.Build.props` file.